kuka_controllers/include/kuka_controllers/joint_impedance_controller.hpp

```diff
@@ -56,6 +58,22 @@ class JointImpedanceController : public controller_interface::ControllerInterfac
   rclcpp::Subscription<std_msgs::msg::Float64MultiArray>::SharedPtr joint_impedance_subscriber_;
   std::vector<double> stiffness_;
   std::vector<double> damping_;
+
+  // Degrees of freedom
+  size_t dof_;
+
+  // Storing command joint names for interfaces
+  std::vector<std::string> command_joint_names_;
+
+  // Parameters from ROS for joint_impedance_controller
+  std::shared_ptr<joint_impedance_controller::ParamListener> param_listener_;
+  joint_impedance_controller::Params params_;
+
+  // Private consts
+  const std::vector<std::string> command_interfaces_param = {"stiffness", "damping"};
+  static constexpr double STIFFNESS_DEFAULT = 10;
```

Comment:
in the sampler and the ecs, the default stiffness is 30, but it probably doesn't matter here

Reply:
@Svastits  whats your opinion?